include/ofpi_arp.h

```diff
@@ -95,12 +105,15 @@ int ofp_arp_init_global(void);
 int ofp_arp_term_global(void);
 int ofp_arp_init_local(void);
 void ofp_arp_term_local(void);
+#ifndef OFP_USE_LIBCK
```

Comment:
Is this necessary? I'd like to avoid ifdefs whenever possible. Btw. when `--enable-libck` is set build fails (not caused by this PR).

```
  CCLD     ofp_test_fragmentation
ofp_test_fragmentation.o: In function `init_suite':
/home/NNN/dev/ofp.git/test/cunit/ofp_test_fragmentation.c:121: undefined reference to `ofp_ipv4_lookup_arp_entry_idx'
collect2: error: ld returned 1 exit status
```

Reply:
ofp_arp_ipv4_insert_entry() has an implementation only in ofp_arp.c. Outside ofp_arp.c, it is used only in ofp_root.c also with "#ifndef OFP_USE_LIBCK". From my point of view, it is more consistent like this.

Reply:
Makes sense.